crates/steel/src/account.rs

```diff
@@ -0,0 +1,160 @@
+// Copyright 2025 RISC Zero, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Types related to account queries.
+pub use revm::primitives::{AccountInfo, Bytecode};
```

Comment:
A Steel account query returns the corresponding `revm::primitives::AccountInfo`. This is a bit closer to the actual (revm) EVM execution, but it does not return the storage root (which is also inaccessible from inside the EVM) and the code is returned as `revm::primitives::Bytecode` instead of just `Bytes`. Alternatively, it may be preferable to define and return our own new type:
```rust
pub struct AccountInfo {
    pub nonce: u64,
    pub balance: U256,
    pub storage_root: B256,
    pub code_hash: B256,
    pub code: Option<Byte>,
}
```

Reply:
OK, I did a little research on this: The problem is the storage_root. Unfortunately, this is not accessible in EVM, so there is no easy way to get it from RPC or the revm DB. The only way is to call an `eth_getProof' just to get this value. This is possible, but would require a bit more code.
So do we think the storage_root is necessary? Otherwise the existing alternative would be sufficient and easier.

Reply:
Let's move on without the storage root them. If there becomes a use case for this, we can revisit, possibly adding a helper function of some sort to specifically get the storage root.